src/main/java/scene/SceneList.java

```diff
@@ -40,21 +45,34 @@ private void resetToIntroductionScene() {
         updateDataFileSceneIndex(STARTING_INDEX_FOR_FILE);
     }
 
+    /**
+     * We return the current scene index.
+     * @return The current scene's index.
+     */
     public int getCurrentSceneIndex() {
         return this.currentSceneIndex;
     }
 
+    /**
+     * We increase the scene number by 1.
+     */
```

Comment:
Javadoc of class members can be specified on a single line as follows: 
/** We increase the scene number by 1. */

And maybe can consider starting the javadocs with a verb. Eg. Increases scene index by 1